driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/SourceRepository.scala

```diff
@@ -7,7 +7,7 @@ import pureconfig.generic.semiauto.deriveReader
 
 sealed trait SourceRepository
 object SourceRepository extends ConfigFormat {
-  final case class Git(path: URI, branch: Option[String]) extends SourceRepository
+  final case class Git(path: URI, ref: Option[String]) extends SourceRepository
```

Comment:
Reader should probably accept any of those: `ref`, `tag`, `branch`, `commit` ?

Reply:
For now I just enabled the use of this for all ref scenarios. There is an issue https://github.com/VirtusLab/ide-probe/issues/20 that should make sure that all git related places have common model and methods.